Adyen/UI/Form/Items/Value Pickers/Abstract/BaseFormPickerItem.swift

```diff
@@ -31,25 +31,40 @@ public struct BasePickerElement<ElementType: CustomStringConvertible>: PickerEle
     }
 
     public var description: String { element.description }
+    
+    public init(identifier: String, element: ElementType) {
+        self.identifier = identifier
+        self.element = element
+    }
 
 }
 
 /// Describes a picker item.
 /// :nodoc:
 open class BaseFormPickerItem<ElementType: CustomStringConvertible>: FormValueItem<BasePickerElement<ElementType>, FormTextItemStyle>,
-    InputViewRequiringFormItem {
+    InputViewRequiringFormItem, Hidable {
 
+    /// :nodoc:
+    public var isHidden: Observable<Bool> = Observable(false)
+    
     /// The complete list of selectable values.
-    internal var selectableValues: [BasePickerElement<ElementType>]
+    @Observable([]) internal var selectableValues: [BasePickerElement<ElementType>]
 
     /// Initializes the picker item.
     ///
     /// - Parameter selectableValues: The list of values to select from.
     /// - Parameter style: The `FormPhoneExtensionPickerItem` UI style.
-    internal init(preselectedValue: BasePickerElement<ElementType>, selectableValues: [BasePickerElement<ElementType>], style: FormTextItemStyle) {
+    public init(preselectedValue: BasePickerElement<ElementType>, selectableValues: [BasePickerElement<ElementType>], style: FormTextItemStyle) {
         assert(selectableValues.count > 0)
-        self.selectableValues = selectableValues
         super.init(value: preselectedValue, style: style)
+        self.selectableValues = selectableValues
+    }
+    
+    /// Updates the selectable values items of the picker
+    /// and triggers the observing view component to update it's pickerview
+    /// - Parameter values: New selectable values to update.
+    public func updateSelectableValues(_ values: [BasePickerElement<ElementType>]) {
+        self.selectableValues = values
     }
```

Comment:
this function seems redundant, since you can call just `BaseFormPickerItem.selectableValues  = newValue`, right?